contracts/ComptrollerStorage.sol

```diff
@@ -137,6 +138,10 @@ contract ComptrollerV4Storage is ComptrollerV3Storage {
 }
 
 contract ComptrollerV5Storage is ComptrollerV4Storage {
+    /**
+    * @notice Is market paused for collateral
+    */
+    mapping(address => uint) public collateralGuardianPaused;
```

Comment:
This introduces a storage slot conflict since it's introduced before another already used slot.

Reply:
thats why you had to create a v6 and v7?

Reply:
It'll be fine as long as the newly introduced ones are after all the already used slots (so technically, if you did it at the end of V5 rather than at the top, it'll be fine too). I introduced V6 just to show that storage structure's been changed.